scripts/obtain/dotnet-install.ps1

```diff
@@ -565,20 +567,30 @@ catch {
         $ZipPath = [System.IO.Path]::combine([System.IO.Path]::GetTempPath(), [System.IO.Path]::GetRandomFileName())
         Say-Verbose "Legacy zip path: $ZipPath"
         Say "Downloading legacy link: $DownloadLink"
-        DownloadFile -Uri $DownloadLink -OutPath $ZipPath
+        try {
+            DownloadFile -Uri $DownloadLink -OutPath $ZipPath
+        }
+        catch {
+            Say "Cannot download: $DownloadLink"
+            $DownloadFailed = $true
+        }
     }
     else {
-        throw "Could not download $assetName version $SpecificVersion"
+        $DownloadFailed = $true
     }
 }
 
+if ($DownloadFailed) {
+    throw "Could not find/download: `"$assetName`" with version = $SpecificVersion`nRefer to: https://github.com/dotnet/core/blob/master/os-lifecycle-policy.md for information on .NET Core support"
```

Comment:
Why this URL? 

@mairaw can we get a better page in. docs and make a mslink to it, pointing temporarily at this location until the new page is ready? 

Regardless of whether we do that, should this be a direct link to GitHub or an mslink?

Reply:
@KathleenDollard 
@leecow provided the URL - if you have a preferred URL; let's discuss.

Reply:
I have created https://aka.ms/dotnet-os-lifecycle, which points to the github page.

Reply:
I think my comment wasn't clear. The download fails: we send them to an LTS page. Hmmm. 

@johnbeisner could we brainstorm a list of reasons we'd fail here: network down? our site is down? We've removed a download? Anything else? 

@mairaw I think a page that lists the possibilities and next steps for the users would be more helpful. At the very least, I want this to be a redirect so we aren't hardcoding to the page that feels wrong. Does a "Why your .NET Core download may have failed" page belong in docs?